source/collections/VectorList.ooc

```diff
@@ -119,9 +124,12 @@ VectorList: class <T> {
 			result add(this[indices[i]])
 		result
 	}
-	getSlice: func (range: Range) -> This<T> {
-		result := This<T> new(range count)
-		this getSliceInto(range, result&)
+	getSlice: func (start, end: Int) -> This<T> {
```

Comment:
I still think that it is awkward to use `range` here, since the last element is usually excluded. But here (as seen in `FloatVectorListTest.ooc `) 
`slice := list getSlice(1 .. 2)` is equivalent to `slice := list getSlice(1, 2)`.

I suggest, I remove `getSlice: func ~range (range: Range)` and replace it with `getSlice: func (start, end: Int)`.